src/utils.cpp

```diff
@@ -1610,6 +1610,42 @@ bool biocyber_compatibility(struct obj_data *obj1, struct obj_data *obj2, struct
           return FALSE;
         }
         break;
+      case CYB_CYBERARMS:
+        if (GET_OBJ_VAL(bio1, 0) == BIO_MUSCLEAUG || GET_OBJ_VAL(bio1, 0) == BIO_MUSCLETONER
+         || GET_OBJ_VAL(bio1, 0) == BIO_ORTHOSKIN || GET_OBJ_VAL(cyber2, 0) == CYB_DERMALPLATING
+         || GET_OBJ_VAL(cyber2, 0) == CYB_DERMALSHEATHING || GET_OBJ_VAL(cyber2, 0) == CYB_BONELACING
+         || GET_OBJ_VAL(cyber2, 0) == CYB_CYBERARMS) {
+          send_to_char("Cyber Arms are incompatible with these augmentations.\r\n", ch);
```

Comment:
As much as it's gonna suck, you should write out the augmentations that it's incompatible with, if only to save yourself the next decade of `newbie which augmentations are incompatible with cyber arms?`

Reply:
also, same thing with magic numbers, use `GET_CYBERWARE_TYPE()`. There's no GET_BIOWARE_TYPE yet, but you can add it in utils.h (search for the cyberware one then stick it in the ITEM_BIOWARE section above it).

Reply:
The builder in me saw the obvious solution being a helpfile explaining the incompatibilities.... But that would require people to read. This is an uncertainty, at best, i'm aware.